packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -247,6 +269,49 @@ export class WaferMap extends FoundationElement {
                 this.renderer.drawWafer();
             }
             this.zoomHandler.connect();
+            if (this.dataManager instanceof ExperimentalDataManager) {
```

Comment:
Can you pull this implementation for the if block into a private method, just for readability and consistency's sake?

Reply:
did it ![image](https://github.com/ni/nimble/assets/110180309/be525aaa-e6d3-4df2-8be7-c44a8400aa07)